libcloud/security.py

```diff
@@ -71,7 +71,7 @@
 
     # If a provided file exists we ignore other common paths because we
     # don't want to fall-back to a potentially less restrictive bundle
-    CA_CERTS_PATH = [environment_cert_file]
```

Comment:
Hm, so this didn't work correctly since 2.0.0?

Reply:
This was working because [`http.LibcloudBaseConnection.setup_ca_cert()`](https://github.com/apache/libcloud/blob/trunk/libcloud/http.py#L147) handles lists by taking the first element and issuing a warning. The only thing that was not working was appending to `libcloud.security.CA_CERTS_PATH`. But defining it as a list is still working.